pysmt/solvers/btor.py

```diff
@@ -191,6 +191,7 @@ def __init__(self, environment, logic, **options):
         self.mgr = environment.formula_manager
         self.declarations = {}
         self._named_assertions = {}
+        self._last_assumptions = []
```

Comment:
Maybe just `assumptions`? We cannot have multiple assumptions at the same time, right? That's why we are satisfied with storing only one set of assumptions.